src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -178,7 +182,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             DockerAccess access = null;
             try {
                 access = createDockerAccess(minimalApiVersion);
-                ServiceHub serviceHub = serviceHubFactory.createServiceHub(project, session, access, log, logSpecFactory);
+                fetchLimit = resolvedImages.get(0).getRunConfiguration().getFetchLimit() != null
```

Comment:
E.g. 

* for fetching by name: since=<container name>&limit=1
* for fetching by images: ancestor=<image name>&limit=1

Hopefully the filter is inclusive. 'will try this out now ...